docs/core/diagnostics/dotnet-counters.md

```diff
@@ -51,7 +51,7 @@ Periodically collect selected counter values and export them into a specified fi
 ### Synopsis
 
 ```console
-dotnet-counters collect [-h|--help] [-p|--process-id] [--refreshInterval] [counter_list] [--format] [-o|--output]
+dotnet-counters collect [-h|--help] [-p|--process-id] [--refreshInterval] [--counters] [counter_list] [--format] [-o|--output] [-- <command>]
```

Comment:
```suggestion
dotnet-counters collect [-h|--help] [-p|--process-id] [--refreshInterval] [--counters] [--format] [-o|--output] [-- <command>]
```
I suggest we document it as if --counters was required to specify the list. If people don't know about the secret back-compatible syntax I don't think they benefit from learning it.

Reply:
```suggestion
dotnet-counters collect [-h|--help] [-p|--process-id] [--refreshInterval] [--counters <COUNTERS>] [--format] [-o|--output] [-- <command>]
```